indexers/social_feed/social_feed.sql

```diff
@@ -1,55 +1,66 @@
 CREATE TABLE
-    "posts" (
-                "id" SERIAL NOT NULL,
-                "account_id" VARCHAR NOT NULL,
-                "block_height" DECIMAL(58, 0) NOT NULL,
-                "receipt_id" VARCHAR NOT NULL,
-                "content" TEXT NOT NULL,
-                "block_timestamp" DECIMAL(20, 0) NOT NULL,
-                "accounts_liked" JSONB NOT NULL DEFAULT '[]',
-                "last_comment_timestamp" DECIMAL(20, 0),
-                CONSTRAINT "posts_pkey" PRIMARY KEY ("id")
-);
+  "posts" (
+    "id" SERIAL NOT NULL,
+    "account_id" VARCHAR NOT NULL,
+    "block_height" DECIMAL(58, 0) NOT NULL,
+    "receipt_id" VARCHAR NOT NULL,
+    "content" TEXT NOT NULL,
+    "block_timestamp" DECIMAL(20, 0) NOT NULL,
+    "accounts_liked" JSONB NOT NULL DEFAULT '[]',
+    "last_comment_timestamp" DECIMAL(20, 0),
+    CONSTRAINT "posts_pkey" PRIMARY KEY ("id")
+  );
 
 CREATE TABLE
-    "comments" (
-                   "id" SERIAL NOT NULL,
-                   "post_id" SERIAL NOT NULL,
-                   "account_id" VARCHAR NOT NULL,
-                   "block_height" DECIMAL(58, 0) NOT NULL,
-                   "content" TEXT NOT NULL,
-                   "block_timestamp" DECIMAL(20, 0) NOT NULL,
-                   "receipt_id" VARCHAR NOT NULL,
-                   CONSTRAINT "comments_pkey" PRIMARY KEY ("id")
-);
+  "comments" (
+    "id" SERIAL NOT NULL,
+    "post_id" SERIAL NOT NULL,
+    "account_id" VARCHAR NOT NULL,
+    "block_height" DECIMAL(58, 0) NOT NULL,
+    "content" TEXT NOT NULL,
+    "block_timestamp" DECIMAL(20, 0) NOT NULL,
+    "receipt_id" VARCHAR NOT NULL,
+    CONSTRAINT "comments_pkey" PRIMARY KEY ("id")
+  );
 
 CREATE TABLE
-    "post_likes" (
-                     "post_id" SERIAL NOT NULL,
-                     "account_id" VARCHAR NOT NULL,
-                     "block_height" DECIMAL(58, 0),
-                     "block_timestamp" DECIMAL(20, 0) NOT NULL,
-                     "receipt_id" VARCHAR NOT NULL,
-                     CONSTRAINT "post_likes_pkey" PRIMARY KEY ("post_id", "account_id")
-);
+  "post_likes" (
+    "post_id" SERIAL NOT NULL,
+    "account_id" VARCHAR NOT NULL,
+    "block_height" DECIMAL(58, 0),
+    "block_timestamp" DECIMAL(20, 0) NOT NULL,
+    "receipt_id" VARCHAR NOT NULL,
+    CONSTRAINT "post_likes_pkey" PRIMARY KEY ("post_id", "account_id")
+  );
+  
+CREATE TABLE
+  "promote" (
```

Comment:
would be more explicit if titled `promoted_post`

Reply:
I think we can promote lots of things, though you are right that we're making use of post_id for everything here.
`promotion_type` is blog for current data.